cranelift/filetests/filetests/isa/aarch64/condbr.clif

```diff
@@ -253,3 +253,357 @@ block1:
 ; check: movz x0, #1
 ; nextln: ldp fp, lr, [sp], #16
 ; nextln: ret
+
+
+function %i128_brz(i128){
+block0(v0: i128):
+    brz v0, block1
+    jump block1
+
+block1:
+    nop
+    return
+}
+
+; check: stp fp, lr, [sp, #-16]!
+; nextln: mov fp, sp
+; nextln: orr x0, x0, x1
+; nextln: cbz x0, label1 ; b label2
```

Comment:
I wonder: would it be better here to do an `ORRS` into `XZR` and then branch on the `Z` flag (here and for brnz below)? That would avoid the need to clobber a register, reducing pressure.

Reply:
I think `ORRS` is no longer an instruction in aarch64, at least I'm not finding it in the [instruction listing](https://developer.arm.com/documentation/dui0802/a/A64-General-Instructions/A64-general-instructions-in-alphabetical-order?lang=en)

Reply:
Ah! I have been deceived by old ARM32 online docs, then; in this case, I agree this is probably the best we can do. Thanks!